CVLByExample/fund-eq-of-dai-certora/certora/conf/runVat.conf

```diff
@@ -0,0 +1,10 @@
+{
+  // files that are part of the scence (eveything Prover can reason about)
+  "files": [
+    "contracts/Vat.sol"
+  ],
+  // the main contract under test and the spec file 
+  "verify": "Vat:certora/specs/Vat.spec",
+  "server": "staging"
+}
+// alternatively, run `certoraRun contracts/BallGame.sol --verify BallGame:certora/specs/BallGame.spec`
```

Comment:
great comments in the conf file - please add in other files as well.
remove the
// alternatively, run `certoraRun contracts/BallGame.sol --verify BallGame:certora/specs/BallGame.spec`